test/Microsoft.ML.TestFramework/BaseTestBaseline.cs

```diff
@@ -96,6 +97,7 @@ protected override void Initialize()
             _baselineCommonDir = Path.Combine(baselineRootDir, "Common");
             _baselineBuildStringDir = Path.Combine(baselineRootDir, BuildString);
             _baselineConfigDirs = GetConfigurationDirs();
+            _usedSpecificBaselineConfigs = new HashSet<string>();
```

Comment:
>_usedSpecificBaselineConfigs [](start = 12, length = 28)

I don't think this is necessary, already covered in _baselineConfigDirs.

You can add any combination to GetConfigurationDirs method and to _baselineConfigDirs.
Say Linux x64 netcoreapp31 then you can add Linux-x64-netcoreapp31 configuration and add baseline file in that configuration folder for specific test if necessary